docs/developer/index.rst

```diff
@@ -516,6 +516,48 @@ parent classes to match. This was not attempted as it would require
 significantly-larger changes to Sherpa (but this change could still be
 made in the future).
 
+.. _pha_filter:
+
+PHA Filtering
+-------------
+
+Filtering of a :py:class:`~sherpa.astro.data.DataPHA` object has two
+complications compared to :py:class:`~sherpa.data.Data1D` objects:
+
+- the independent axis can be referred to in channel units (normally 1
+  to the maximum number of channels), energy units (e.g. 0.5 to 7
+  keV), or wavelength units (e.g. 20 to 22 Angstroms);
+
+- each channel has a width of 1, so channel filters - which are
+  generally going to be integer values - map exactly, but each channel
+  has a finite width in the derived units (that is, energy or
+  wavelength) so multiple values will map to the same channel;
+
+- the data can be dynamically grouped via the
+  :py:attr:`~sherpa.astro.data.DataPHA.grouping` attribute, normally set
+  by methods like :py:meth:`~sherpa.astro.data.DataPHA.group_counts` and
+  controlled by the :py:meth:`~sherpa.astro.data.DataPHA.group` method,
+  which means that the desired filter, when mapped to channel units,
+  is likely to end up partially overlapping the first and last groups,
```

Comment:
end on comma. Something cut off?

Reply:
I have changed this section (added more text for this point and added a fourth point to the list).